Convert failed when GPU driver version is not 410.104.0

## Error logs
The log of `make test`.
```
# Run Blueoil test of classification
CUDA_VISIBLE_DEVICES=0 bash ./blueoil_test.sh  --task classification
Creating directory for test: ./tmp/tests/20190913175132

# Basic tests
## Test of caltech101_classification
### ./blueoil.sh init
### ls config/created_by_test_script_caltech101_classification.yml
OK!
### diff ./tmp/tests/20190913175132/created_by_test_script_caltech101_classification.yml ./tests/config/caltech101_classification.yml
OK!
### ./blueoil.sh train ./tmp/tests/20190913175132/created_by_test_script_caltech101_classification.yml
OK!
### ./blueoil.sh convert ./tmp/tests/20190913175132/created_by_test_script_caltech101_classification.yml ./saved/created_by_test_script_caltech101_classification_20190913175136
##############################
ERROR result is 1 (expect 0)
##############################
#################################################
Erorr log : ./tmp/tests/20190913175132/test_3.log
#################################################
### ./blueoil.sh convert ./tmp/tests/20190913175132/created_by_test_script_caltech101_classification.yml ./saved/created_by_test_script_caltech101_classification_20190913175136
#### Generate output files ####

WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.

WARNING:tensorflow:From /usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
----------------- config value --------------------
{'BATCH_SIZE': 1,
 'CLASSES': ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'],
 'DATASET': {'AUGMENTOR': [Blur({'min_value': 0, 'max_value': 1})],
             'BATCH_SIZE': 1,
             'DATA_FORMAT': 'NHWC',
             'PRE_PROCESSOR': [Resize({'size': [128, 128]}), DivideBy255({})]},
 'DATASET_CLASS': <class 'abc.DATASET_CLASS'>,
 'DATA_FORMAT': 'NHWC',
 'IMAGE_SIZE': [128, 128],
 'IS_DEBUG': False,
 'IS_PRETRAIN': False,
 'KEEP_CHECKPOINT_MAX': 5,
 'MAX_EPOCHS': 1,
 'NETWORK': {'ACTIVATION_QUANTIZER': <function linear_mid_tread_half_quantizer at 0x7f17f9b148c8>,
             'ACTIVATION_QUANTIZER_KWARGS': {'bit': 2, 'max_value': 2},
             'BATCH_SIZE': 1,
             'DATA_FORMAT': 'NHWC',
             'IMAGE_SIZE': [128, 128],
             'LEARNING_RATE_FUNC': None,
             'LEARNING_RATE_KWARGS': None,
             'OPTIMIZER_CLASS': <class 'tensorflow.python.training.adam.AdamOptimizer'>,
             'OPTIMIZER_KWARGS': {'learning_rate': 0.001},
             'WEIGHT_DECAY_RATE': 0.0005,
             'WEIGHT_QUANTIZER': <function binary_mean_scaling_quantizer at 0x7f17f9b149d8>,
             'WEIGHT_QUANTIZER_KWARGS': {}},
 'NETWORK_CLASS': <class 'lmnet.networks.classification.lmnet_v1.LmnetV1Quantize'>,
 'POST_PROCESSOR': None,
 'PRETRAIN_DIR': '',
 'PRETRAIN_FILE': '',
 'PRETRAIN_VARS': [],
 'PRE_PROCESSOR': [Resize({'size': [128, 128]}), DivideBy255({})],
 'SAVE_CHECKPOINT_STEPS': 1000,
 'SUMMARISE_STEPS': 100,
 'TASK': <Tasks.CLASSIFICATION: 'IMAGE.CLASSIFICATION'>,
 'TEST_STEPS': 1000}
----------------- config value --------------------
Restore from /home/blueoil/saved/created_by_test_script_caltech101_classification_20190913175136/checkpoints/save.ckpt-27
WARNING:tensorflow:From /home/blueoil/lmnet/lmnet/blocks.py:92: conv2d (from tensorflow.python.layers.convolutional) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.conv2d instead.
WARNING:tensorflow:From /usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/tensorflow/python/framework/function.py:1007: calling Graph.create_op (from tensorflow.python.framework.ops) with compute_shapes is deprecated and will be removed in a future version.
Instructions for updating:
Shapes are always computed; don't use the compute_shapes as it has no effect.
WARNING:tensorflow:From /home/blueoil/lmnet/lmnet/networks/classification/lmnet_v1.py:86: dropout (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.dropout instead.
WARNING:tensorflow:From /home/blueoil/lmnet/lmnet/networks/classification/lmnet_v1.py:107: average_pooling2d (from tensorflow.python.layers.pooling) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.average_pooling2d instead.
2019-09-13 08:52:06.740570: E tensorflow/stream_executor/cuda/cuda_driver.cc:300] failed call to cuInit: CUDA_ERROR_SYSTEM_DRIVER_MISMATCH: system has unsupported display driver / cuda driver combination
2019-09-13 08:52:06.740860: E tensorflow/stream_executor/cuda/cuda_diagnostics.cc:306] kernel version 418.67.0 does not match DSO version 410.104.0 -- cannot find working devices in this configuration
WARNING:tensorflow:From /usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/tensorflow/python/training/saver.py:1266: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
WARNING:tensorflow:From /home/blueoil/lmnet/lmnet/utils/executor.py:94: convert_variables_to_constants (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.compat.v1.graph_util.convert_variables_to_constants
WARNING:tensorflow:From /usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/tensorflow/python/framework/graph_util_impl.py:245: extract_sub_graph (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.compat.v1.graph_util.extract_sub_graph
Traceback (most recent call last):
  File "/usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1334, in _do_call
    return fn(*args)
  File "/usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1319, in _run_fn
    options, feed_dict, fetch_list, target_list, run_metadata)
  File "/usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1407, in _call_tf_sessionrun
    run_metadata)
tensorflow.python.framework.errors_impl.OutOfRangeError: Node 'conv1/BatchNorm/FusedBatchNorm' (type: '_FusedConv2D', num of outputs: 1) does not have output 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "blueoil/blueoil_convert.py", line 173, in <module>
    main()
  File "/usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "blueoil/blueoil_convert.py", line 169, in main
    run(experiment_id, restore_path)
  File "blueoil/blueoil_convert.py", line 118, in run
    export_dir = run_export(experiment_id, restore_path=restore_path)
  File "/home/blueoil/lmnet/executor/export.py", line 199, in run
    return _export(config, restore_path, image)
  File "/home/blueoil/lmnet/executor/export.py", line 138, in _export
    val = sess.run(op_output.name, feed_dict=feed_dict)
  File "/usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 929, in run
    run_metadata_ptr)
  File "/usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1152, in _run
    feed_dict_tensor, options, run_metadata)
  File "/usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1328, in _do_run
    run_metadata)
  File "/usr/local/pyenv/versions/python3.6/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1348, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.OutOfRangeError: Node 'conv1/BatchNorm/FusedBatchNorm' (type: '_FusedConv2D', num of outputs: 1) does not have output 1
ERROR: Failed to generate output files

Move files created by tests to ./tmp/tests/20190913175132
If you want to clean up files created by tests, you can run 'rm -rf ./tmp/tests/20190913175132'
Makefile:22: recipe for target 'test-classification' failed
make: *** [test-classification] Interrupt
```

This error causes when tensorflow is CPU mode, related #106 
I checked and found the tensorflow on our docker image is not able to use GPU correctly.
```
2019-09-13 09:18:21.430386: E tensorflow/stream_executor/cuda/cuda_driver.cc:300] failed call to cuInit: CUDA_ERROR_UNKNOWN: unknown error
2019-09-13 09:18:21.430567: E tensorflow/stream_executor/cuda/cuda_diagnostics.cc:306] kernel version 410.104.0 does not match DSO version 410.48.0 -- cannot find working devices in this configuration
```

## Why it occurred?
This is caused by the following line:
https://github.com/blue-oil/blueoil/blob/master/docker/Dockerfile#L126

We should not manage cuda driver paths and should entrust them to nvidia-runtime.
I will fix it.

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -123,10 +123,11 @@ ENV HOROVOD_GPU_ALLREDUCE NCCL
 ENV HOROVOD_WITH_TENSORFLOW 1
 
 ENV CUDA_HOME /usr/local/cuda-10.0
-ENV LD_LIBRARY_PATH /usr/local/cuda-10.0/compat/:/usr/local/cuda-10.0/targets/x86_64-linux/lib/:$LD_LIBRARY_PATH
 
-# Install requirements for distributed training
-RUN pip install -r /tmp/requirements/dist.requirements.txt
+# Install requirements for distributed training temporarily using CUDA stubs
+RUN ldconfig /usr/local/cuda-10.0/targets/x86_64-linux/lib/stubs && \
+    pip install -r /tmp/requirements/dist.requirements.txt && \
+    ldconfig
 
 # Configure OpenMPI to run good defaults:
 #   --bind-to none --map-by slot --mca btl_tcp_if_exclude lo,docker0 --mca btl_vader_single_copy_mechanism none

